lib/review/review.controllers.js

```diff
@@ -1,53 +1,210 @@
 module.exports = (models) => {
   const Boom = require('boom');
   const reviewModels = require('./review.models');
+  const accountLib = require('../account/account.controllers')(models).lib;
```

Comment:
any time 'owned' content is created we will need to import the account module and make sure the user has an account. This may eventually be abstracted into an auth strategy.